lib/Specification.cpp

```diff
@@ -326,8 +338,11 @@ Specification::DecodeFromPB(llvm::LLVMContext &context, const std::string &pb) {
     return arch.Error();
   }
 
+  auto image_name = spec.image_name();
```

Comment:
One more copy here which you can avoid with `const auto&`.

Reply:
[ccbac6a](https://github.com/lifting-bits/anvill/pull/379/commits/ccbac6a4749f767dabadc244e8739256f5c5e46a)